docs/typescript/virtuals.md

```diff
@@ -1,6 +1,34 @@
 # Virtuals in TypeScript
 
 [Virtuals](/docs/tutorials/virtuals.html) are computed properties: you can access virtuals on hydrated Mongoose documents, but virtuals are **not** stored in MongoDB.
+Mongoose supports auto typed virtuals so you don't need to define additional typescript interface anymore but you are still able to do so.
+
+### 1- Auto types:
```

Comment:
Minor note: no need for `1- ` here, that isn't consistent with the rest of our docs. Just call this section "Automatically Inferred Types"

Reply:
I will refactor that soon

Reply:
done